workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx

```diff
@@ -219,6 +219,7 @@ const DealRegistrationList = () => {
 		{
 			component: (
 				<DateFilter
+					clearInputs={!filters?.dataSubmitted?.dates.startDate?.length && !filters?.dataSubmitted?.dates.endDate?.length}
```

Comment:
@eduardo-diniz , would it be possible to do this check in the useEffect() inside of DateFilter.ts? May save us some variable/property setting and passing if we can, and make things a lot simplier and cleaner

Reply:
Yes, sure
I send `filters?.['previousPropertyDate']` to DateFilter.ts, could you check the PR again pls?